test/test_autograd.py

```diff
@@ -4653,18 +4654,21 @@ def run_functional_checks(test_case, test_name, name, apply_fn, run_grad_checks,
 # the tests for these ops which do not have 'complex' in variant should not run for complex
 # and only run for floating point
 
-separate_complex_tests = ['log', 'log10', 'log1p', 'log2', 'reciprocal', 'tan']
+# TODO: add the commented tests back after updating the formula based on tensorflow definition - @anjali411
```

Comment:
nit: A convenient way to tag TODOs is to to `TODO(anjali411)` so that you search for that to get all the todos in the repo